tests/python/gaia-ui-tests/gaiatest/visual_components/visual_component.py

```diff
@@ -0,0 +1,7 @@
+from gaiatest.apps.base import PageRegion
+
+
+class VisualComponent(PageRegion):
```

Comment:
Wat is a VisualComponent? Isn't everything a VisualComponent? I mean, can we have name that better describes what it is? To me, almost everything can be a visual component.